nilerror/go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/rivercontrib/nilerror
+
+go 1.24.2
+
+replace github.com/riverqueue/river => ../../river
+
+replace github.com/riverqueue/river/riverdriver => ../../river/riverdriver
+
+replace github.com/riverqueue/river/riverdriver/riverpgxv5 => ../../river/riverdriver/riverpgxv5
+
+replace github.com/riverqueue/river/rivershared => ../../river/rivershared
+
+replace github.com/riverqueue/river/rivertype => ../../river/rivertype
```

Comment:
Note to not merge this before nuking these replaces and instead pointing at the upcoming River release